OPLS.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python
 
 ### Author:
-# Daisy Y. Kyu
 # Nam Thi
-# Victoria Lim (limvt@uci.edu)
 # Caitlin C. Bannan (bannanc@uci.edu)
+# Daisy Y. Kyu
```

Comment:
I didn't actually add her, she was already on the list, I just changed the order of authors.